source/Coop.Core/Client/Services/MobileParties/Messages/SettlementEnterRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Coop.Core.Client.Services.MapEvents.Messages
```

Comment:
The `MapEvents` service shouldn't exist. This is all within the scope of a MobileParty so it should exist in the `MobileParties` service